paddle/gserver/layers/ConcatenateLayer.cpp

```diff
@@ -110,6 +112,8 @@ class ConcatenateLayer2 : public Layer {
   std::vector<std::unique_ptr<Projection>> projections_;
   std::vector<Argument> projOutput_;
   std::vector<std::pair<size_t, size_t>> projCol_;
+  bool isConvProj_;
```

Comment:
Use sharedBiases_ here, and configured to True in config_parser.py. 
Than ConcatenateLayer2 do not need to know if the input projections_ is ConvProjection.